nuget/AppCenter.nuspec

```diff
@@ -36,8 +36,8 @@
   <files>
 
     <!-- Portable -->
-    <file src="$pcl_dir$/Microsoft.AppCenter.dll" target="lib/portable-net45+win8+wpa81+wp8" />
-    <file src="$pcl_dir$/Microsoft.AppCenter.xml" target="lib/portable-net45+win8+wpa81+wp8" />
+    <!--<file src="$pcl_dir$/Microsoft.AppCenter.dll" target="lib/portable-net45+win8+wpa81+wp8" />
```

Comment:
What is this change? We still need to support PCL in nuget.

Reply:
This is me trying different approaches. Pay no attention, this is clearly wrong.

Reply:
Github tricked me into thinking it was current change. My comment was instantly marked as outdated :D

Happens all the time with notifications.